src/core/build_target.go

```diff
@@ -31,6 +31,10 @@ const DefaultBuildingDescription = "Building..."
 const buildDirSuffix = "._build"
 const testDirSuffix = "._test"
 
+// TmpOutputFormat is a Temporary formatting for outputs, this is used to avoid name conflicts
+// when package directory name and output are the same
+const tmpOutputFormat = "%s.out"
```

Comment:
probably just use this inside the function, don't think it needs a separate constant.